packages/utils/tokens/src/border-tokens.ts

```diff
@@ -9,13 +9,15 @@ export interface IBorderTokens {
   borderWidth?: number;
   borderRadius?: number;
   borderStyle?: ViewStyle['borderStyle'];
+  borderBottomWidth?: ViewStyle['borderBottomWidth'];
 }
 
 export const borderTokens: IOperationSet<IBorderTokens, ITheme> = [
   { source: 'borderColor', lookup: getPaletteFromTheme },
   { source: 'borderWidth' },
   { source: 'borderRadius' },
   { source: 'borderStyle' },
+  { source: 'borderBottomWidth' },
 ];
 
-export const borderStyles = tokenBuilder<IBorderTokens>('borderColor', 'borderRadius', 'borderStyle', 'borderWidth');
+export const borderStyles = tokenBuilder<IBorderTokens>('borderColor', 'borderRadius', 'borderStyle', 'borderWidth'); //'borderBottomWidth');
```

Comment:
Why is this commented out?

Reply:
No reason at all, just an accident from when we were debugging w/ krystal earlier